src/SDKs/SqlManagement/Sql.Tests/CapabilitiesScenarioTests.cs

```diff
@@ -40,6 +40,12 @@ public void TestGetCapabilities()
                                 Assert.NotNull(m.Unit);
                             }
                         }
+                        Assert.Null(e.ZoneRedundant);
```

Comment:
when code hits production, this will no longer be null, right? probably better to just remove these asserts for now.